examples/direct_fidelity_estimation.py

```diff
@@ -0,0 +1,82 @@
+"""Implements direct fidelity estimation.
+
+Direct Fidelity Estimation from Few Pauli Measurements
+https://arxiv.org/abs/1104.4695
+
+Practical characterization of quantum devices without tomography
+https://arxiv.org/abs/1104.3835
+"""
+
+import cirq
+import heapq
+import itertools
+import math
+import numpy
+
+
+def build_circuit():
+  # Builds an arbitrary circuit to test. The circuit is non Clifford to show the
+  # use of simulators.
+  qubits = cirq.LineQubit.range(3)
+  circuit = cirq.Circuit(
+      cirq.Z(qubits[0])**0.25,  # T-Gate, non Clifford.
+      cirq.X(qubits[1])**0.123,
+      cirq.X(qubits[2])**0.456)
+  return circuit, qubits
+
+
+def compute_characteristic_function(circuit, P_i, qubits, noise):
+  n = len(P_i)
+  d = 2**n
+
+  trace = 0  # rho_i / sigma_i in https://arxiv.org/pdf/1104.3835.pdf
```

Comment:
does this comment mean "rho_i *or* sigma_i" or "rho_i divided by sigma_i". I think the former(?) but both expressions are present in the linked paper!

Reply:
Sorry, it was indeed a confusing way to write things. It is 'or'.